x/token/keeper/querier.go

```diff
@@ -37,14 +45,29 @@ func queryBalance(ctx sdk.Context, path []string, _ abci.RequestQuery, keeper Ke
 		return nil, sdkerrors.Wrap(sdkerrors.ErrInvalidAddress, err.Error())
 	}
 
-	balance := utils.TokenToFloat64(keeper.GetBalance(ctx, owner))
+	bz, err := keeper.cdc.MarshalBinaryBare(Balance{
+		Balance:      keeper.GetBalance(ctx, owner).ToDec().QuoInt64(types.Denominator),
+		BalanceDelta: keeper.GetBalanceDelta(ctx, owner).ToDec().QuoInt64(types.Denominator),
+	})
+	if err != nil {
+		return nil, sdkerrors.Wrap(sdkerrors.ErrJSONMarshal, err.Error())
+	}
+
+	return bz, nil
+}
+
+// nolint: unparam
+func queryPool(ctx sdk.Context, _ []string, _ abci.RequestQuery, keeper Keeper) ([]byte, error) {
+	interval := keeper.GetParams(ctx).RewardsBlockInterval
 
-	res, err := codec.MarshalJSONIndent(keeper.cdc, Balance{
-		Balance: balance,
+	bz, err := keeper.cdc.MarshalBinaryBare(Pool{
+		Size:                       keeper.distributionKeeper.GetFeePoolCommunityCoins(ctx),
```

Comment:
What is the decentrd command to update FeePoolCommunityCoins? It is populated as transaction fee is applied as well?

Reply:
FundCommunityPool as I remember.

Yes, there is a parameter of distribution module 'community_tax' that means how make stakes should be taken